apps/server/src/services/__tests__/timerUtils.test.ts

```diff
@@ -858,7 +858,8 @@ describe('getRuntimeOffset()', () => {
     expect(offsetAbs).toBe(0);
   });
 
-  it('handles loaded event', () => {
+  //TODO: do we actually what to give offsets just on load
```

Comment:
@cpvalente 
what do you think of this
is there a value of giving an offset event if the event is just loaded?